docs/iris/src/userguide/interpolation_and_regridding.rst

```diff
@@ -176,8 +176,8 @@ For example, to mask values that lie beyond the range of the original data:
    >>> scheme = iris.analysis.Linear(extrapolation_mode='mask')
    >>> new_column = column.interpolate(sample_points, scheme)
    >>> print(new_column.coord('altitude').points)
-   [           nan   494.44451904   588.88891602   683.33325195   777.77783203
-      872.222229     966.66674805  1061.11108398  1155.55541992            nan]
+   [-- 494.44451904296875 588.888916015625 683.333251953125 777.77783203125
```

Comment:
You are right about the point you are making, but I don't believe that this is the right time to be making it.  

This PR is to merge a feature branch into Iris which has been under construction for 4 months, and every decision has been discussed in great detail already.  This method may not be ideal, but with dask having no support for masked values it is the best option we have.  

We have by no means kept development of the feature branch a secret, and there has been plenty of time and space for discussion of major implementation decisions, which is not in this PR.  This is just to review the last 10 commits, as @bjlittle pointed out in his first comment, so even though you are right, there is really nothing we can do about it now.